### PR TITLE
Contributing - Correct GH Issue form links

### PR DESCRIPTION
**Pull request by gundalow**
link to `/issues/new/choose` so people can take advantage of the new issue forms

  * [X] Your changes are not possible to do through a plugin and relevant
    to a large audience (ideally all users of OctoPrint)
  * [X] If your changes are large or otherwise disruptive: You have
    made sure your changes don't interfere with current development by
    talking it through with the maintainers, e.g. through a
    Brainstorming ticket
  * [X] Your PR targets OctoPrint's devel branch if it's a completely
    new feature, or maintenance if it's a bug fix or improvement of
    existing functionality for the current stable version (no PRs
    against master or anything else please)
  * [X] Your PR was opened from a custom branch on your repository
    (no PRs from your version of master, maintenance or devel please),
    e.g. dev/my_new_feature or fix/my_bugfix
  * [X] Your PR only contains relevant changes: no unrelated files,
    no dead code, ideally only one commit - rebase and squash your PR
    if necessary!
  * [X] Your changes follow the existing coding style
  * [x] If your changes include style sheets: You have modified the
    .less source files, not the .css files (those are generated with
    lessc)
  * [x] You have tested your changes (please state how!) - ideally you
    have added unit tests
  * [x] You have run the existing unit tests against your changes and
    nothing broke
  * [ ] You have added yourself to the AUTHORS.md file :)

#### How was it tested? How can it be tested by the reviewer?
Viewed the MD file

### CODE REVIEW

**Comment by gundalow** [comment]
@foosel Follow on from `Show off your Issue Forms`

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -126,7 +126,7 @@ there.
 First of all make sure your use **a descriptive title**. "It doesn't work"
 and similar unspecific complaints are NOT descriptive titles.
 
-Always fill out the [reporting form](https://github.com/OctoPrint/OctoPrint/issues/new?template=bug_report.yml) **completely** and include as much
+Always fill out the [reporting form](https://github.com/OctoPrint/OctoPrint/issues/new/choose) **completely** and include as much
```
Comment (cp2004) [inline CONTRIBUTING.md:129]: Why does this need to be changed? The old link links directly to the form, which makes sense in this context, surely?

Comment (gundalow) [reply CONTRIBUTING.md]: Hum, you are right, not sure why I changed that, reverted.

**Comment by gundalow** [comment]
ready for review

**Comment by foosel** [comment]
Thanks! Looks like I missed some!

**Pull request merged**